### PR TITLE
1.3.6 release notes

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: content/en/news/releases/1.3.x/announcing-1.3.6/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+description: Istio 1.3.5 release announcement.
+publishdate: 2019-12-10
+attribution: The Istio Team
+subtitle: Minor Update
+release: 1.3.6
+aliases:
+    - /news/announcing-1.3.6
+---
+
+This release contains fixes for the security vulnerability described in [our December 10, 2019 news post]()  as well as bug fixes to improve robustness. This release note describes what's different between Istio 1.3.5 and Istio 1.3.6.
```
Comment (tanjunchen) [inline content/en/news/releases/1.3.x/announcing-1.3.6/index.md]: [our December 10, 2019 news post]()
maybe need url.

Comment (brian-avery) [reply content/en/news/releases/1.3.x/announcing-1.3.6/index.md]: Thanks. The date was announced but the article isn't up yet. Will update that when it's available.

Comment (tanjunchen) [reply content/en/news/releases/1.3.x/announcing-1.3.6/index.md]: done

Comment (adammil2000) [reply content/en/news/releases/1.3.x/announcing-1.3.6/index.md]: Is the security update section awaiting an entry?

File: content/en/news/releases/1.3.x/announcing-1.3.6/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Announcing Istio 1.3.6
+description: Istio 1.3.5 release announcement.
```
Comment (rlenglet) [inline content/en/news/releases/1.3.x/announcing-1.3.6/index.md]:
```suggestion
description: Istio 1.3.6 release announcement.
```

File: content/en/news/releases/1.3.x/announcing-1.3.6/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+publishdate: 2019-12-10
+attribution: The Istio Team
+subtitle: Minor Update
```
Comment (geeknoid) [inline content/en/news/releases/1.3.x/announcing-1.3.6/index.md]:
```suggestion
subtitle: Patch Release
```

File: content/en/news/releases/1.3.x/announcing-1.3.6/index.md (hunk 2 above)
Comment (geeknoid) [reply content/en/news/releases/1.3.x/announcing-1.3.6/index.md]:
```suggestion
description: Istio 1.3.6 patch release.
```

Please look at the current 1.3.5 release announcement for a current example of what to include in the front-matter. Looks like you copied an older version of the markdown file.

File: content/en/news/releases/1.3.x/announcing-1.3.6/index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: Announcing Istio 1.3.6
+description: Istio 1.3.5 release announcement.
+publishdate: 2019-12-10
+attribution: The Istio Team
```
Comment (geeknoid) [inline content/en/news/releases/1.3.x/announcing-1.3.6/index.md]: Remove the attribution line.

File: content/en/news/releases/1.3.x/announcing-1.3.6/index.md (hunk 2 above)
Comment (brian-avery) [reply content/en/news/releases/1.3.x/announcing-1.3.6/index.md]: Thanks. Updated to the new style.

File: content/en/news/releases/1.3.x/announcing-1.3.6/index.md (hunk 4 above)
Comment (brian-avery) [reply content/en/news/releases/1.3.x/announcing-1.3.6/index.md]: Updated to the new style.

File: content/en/news/releases/1.3.x/announcing-1.3.6/index.md (hunk 3 above)
Comment (brian-avery) [reply content/en/news/releases/1.3.x/announcing-1.3.6/index.md]: Thanks. Updated to the new style.

File: content/en/news/releases/1.3.x/announcing-1.3.6/index.md (hunk 1 above)
Comment (brian-avery) [reply content/en/news/releases/1.3.x/announcing-1.3.6/index.md]: Yep. 1.3.6 will include the issues mentioned in this announcement: 

https://discuss.istio.io/t/upcoming-security-releases-of-istio-1-2-10-1-3-6-and-1-4-1/4553

**Comment by geeknoid** [comment]
I presume this PR is obsolete, since there is currently 1.3.6 release notes on the site?

**Comment by rlenglet** [comment]
Right.

**Pull request closed without merging**